tool_quality_checks/test/test_db.py

```diff
@@ -85,6 +85,9 @@ def tearDownClass(cls):
         cls.db.conn.close()
         drop_database(cls.settings)
 
+    def test_00_empty_database(self):
+        self.db.empty_database()
```

Comment:
Prima om voor dit soort tests geen "assert" ofzo te hebben. Het is soms ook gewoon voldoende dat de methode wordt aangeroepen en zonder foutmelding z'n werk doet.

Ter info: dit wordt een "smoke test" genoemd. Stekker in het stopcontact en kijken of er in ieder geval geen rook uit het apparaat komt. :-)
Bij mijn code vind je nogal eens een `def test_smoke()`, dan weet je waar dat vandaan komt.